azext_edge/tests/edge/init/int/test_init_int.py

```diff
@@ -44,17 +44,16 @@ def init_test_setup(settings, tracked_resources):
         "--enable-hierarchical-namespace --public-network-access Disabled "
         "--allow-shared-key-access false --allow-blob-public-access false --default-action Deny"
     )
-    tracked_resources.append(storage_account['id'])
+    tracked_resources.append(storage_account["id"])
     registry = run(
         f"az iot ops schema registry create -n {registry_name} -g {settings.env.azext_edge_rg} "
-        f"--rn {registry_namespace} --sa-resource-id {storage_account['id']} "
+        f"--rn {registry_namespace} --sa-resource-id {storage_account['id']}"
     )
     tracked_resources.append(registry["id"])
 
     if not all([settings.env.azext_edge_cluster, settings.env.azext_edge_rg]):
         raise AssertionError(
-            "Cannot run init tests without a connected cluster and resource group. "
-            f"Current settings:\n {settings}"
+            "Cannot run init tests without a connected cluster and resource group. " f"Current settings:\n {settings}"
```

Comment:
if you are going to do this then at least combine the strings...

Reply:
sorry i didn't notice - that was black auto-formatting, will adjust some of these